docs/network/interfaces_and_networks.md

```diff
@@ -91,61 +92,56 @@ definition.
 apiVersion: "k8s.cni.cncf.io/v1"
 kind: NetworkAttachmentDefinition
 metadata:
-  name: bridge-test
+  name: linux-bridge-net-ipam
 spec:
   config: '{
       "cniVersion": "0.3.1",
-      "name": "bridge-test",
-      "type": "bridge",
-      "bridge": "br1",
+      "name": "mynet",
       "disableContainerInterface": true
+      "plugins": [
+        {
+          "type": "bridge",
+          "bridge": "br10",
+          "vlan": 0,
+          "ipam": {
+            "type": "whereabouts",
+            "range": "10.1.1.0/24"
+          },
+          "macspoofchk": true
+        }
+      ]
     }'
 ```
 
 With following definition, the VMI will be connected to the default pod
-network and to the secondary Open vSwitch network.
+network and to the secondary bridge network, referencing the `NetworkAttachmentDefinition` 
+shown above(in the same namespace)
 
 ```yaml
 kind: VM
```

Comment:
The partial yaml manifest is broken.
Please consider fixing it.

Please note that this was probably meant to be a `VirtualMachineInstance` manifest.
Please consider converting it to be a `VirtualMachine` manifest.

Reply:
same as above - done